controllers/tekton/pipelinerun_controller.go

```diff
@@ -106,35 +98,35 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 }
 
 // SetupWithManager sets up the controller with the Manager.
-func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
+func (r *PipelineRunReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&devopsv2alpha1.PipelineRun{}).
 		Complete(r)
 }
 
 // deleteExternalResources deletes any external resources associated with the devopsv2alpha1.Pipeline
-func (r *Reconciler) deleteExternalResources(ctx context.Context, pipelineRun *devopsv2alpha1.PipelineRun) error {
+func (r *PipelineRunReconciler) deleteExternalResources(ctx context.Context, pipelineRun *devopsv2alpha1.PipelineRun) error {
 	tknPipelineRunName := pipelineRun.Spec.Name
 	klog.Infof("PipelineRun [%s] is under deletion.", tknPipelineRunName)
 
-	// Find and delete the related Tekton TaskRuns
-	if _, err := r.TknClient.TektonV1beta1().
+	// We will first find the target Tekton PipelineRun CRD resources in the given
+	// namespace. If we do not find it, we will return directly.
+	if _, err := r.TknClientset.TektonV1beta1().
 		PipelineRuns(pipelineRun.Namespace).
 		Get(ctx, tknPipelineRunName, metav1.GetOptions{}); err != nil {
-		klog.Infof("PipelineRun [%s] was not found in namespace %s.", tknPipelineRunName, pipelineRun.Namespace)
-		// Tekton PipelineRun resource does not exist.
-		// We do nothing here.
+		// Tekton PipelineRun resource does not exist, so we just do nothing here.
+		klog.Infof("unable to find Tekton PipelineRun [%s] in namespace %s", tknPipelineRunName, pipelineRun.Namespace)
```

Comment:
klog.Infof -> klog.Errorf

Reply:
> klog.Infof -> klog.Errorf

It's not an error message, you can see that below we return a nil.

Reply:
IMO, the message could be logged as Trace level. So I suggest using `klog.V(5).Infof("...")` instead. See also: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md

Reply:
agree with @JohnNiang